packages/widgets/src/components/TradingScoreboard/TradingScoreboard.stories.tsx

```diff
@@ -0,0 +1,141 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { css } from '@emotion/react';
+import { Icon } from '@ssa-ui-kit/core';
+import { TradingInfoCard } from '@components/TradingInfoCard';
+import TradingScoreboard from './TradingScoreboard';
+import { ITradingScoreboardProps } from './types';
+
+export default {
+  title: 'Widgets/TradingScoreboard',
+  component: TradingScoreboard,
+} as Meta<typeof TradingScoreboard>;
+
+export const Default: StoryObj<typeof TradingScoreboard> = (
+  args: ITradingScoreboardProps,
+) => {
+  return (
+    <TradingScoreboard
+      items={args.items}
+      itemsPerRow={args.itemsPerRow}
+      onClick={(item) => console.log(item)}
```

Comment:
Please add something that will be visible for the user to this handler. E.g. `alert()`.

Reply:
Done